magefiles/target/sync-chart-md.go

```diff
@@ -7,46 +7,42 @@ import (
 	"os"
 	"strings"
 
+	"botkube.io/tools/printer"
 	"github.com/MakeNowJust/heredoc/v2"
 	"github.com/samber/lo"
-	"github.com/tidwall/gjson"
-
-	"botkube.io/tools/printer"
 )
 
 var fileTpl = heredoc.Doc(`
      ---
      id: helm-chart-parameters
      title: Helm chart parameters
-     sidebar_position: 6
+     sidebar_position: 5
      ---
      %s
      `)
 
 const (
-	urlLastCommit     = "https://api.github.com/repos/kubeshop/botkube/commits?per_page=1"
 	urlReadmeBySHAFmt = "https://raw.githubusercontent.com/kubeshop/botkube/%s/helm/botkube/README.md"
 	urlValuesBySHAFmt = "https://github.com/kubeshop/botkube/blob/%s/helm/botkube/values.yaml"
 	dstFilePath       = "docs/configuration/helm-chart-parameters.md"
 )
 
 func SyncChartParams() {
-	printer.Title("Synchronizing Helm chart doc...")
+	printer.Title("Synchronizing Helm chart doc ...")
 
-	lastCommitJSON := getBody(urlLastCommit)
-	sha := gjson.Get(lastCommitJSON, "0.sha").String()
+	botkubeReleaseBranch := os.Getenv("BOTKUBE_RELEASE_BRANCH")
```

Comment:
we use that also on daily basis when we sync helm params for the latest version of docs. Please create a dedicated target or do the release update based on additional flag/arg/env 🙂

Reply:
I couldn't see that mage build in other gh actions, so you run that locally also?

Reply:
yes, it's run locally, when you create a PR :)

Reply:
ok handled, so I use release branch once it is provided, I use commit otherwise
![Screen Shot 2022-10-19 at 18 48 37](https://user-images.githubusercontent.com/1237982/196741619-62750e43-967c-43c7-8b86-87bd1ecf0a39.png)